datacats/cli/create.py

```diff
@@ -20,7 +20,7 @@ def create(opts):
     """Create a new environment
 
 Usage:
-  datacats create [-bin] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
+  datacats create [-bin] [-c NAME] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
```

Comment:
Well, in the use case where you want unique names for each of your sub-environments. That was the use case I was thinking of, but I can see the merit of doing it in "init" also.